README.md

```diff
@@ -101,6 +101,11 @@ support for creating testing mocks.
 This Python library can be installed with <code>pip</code>. It provides
 support for advanced functools such as OrderedDict and lru_cache memoization.
 
+* pytz (https://pypi.org/project/pytz)
```

Comment:
We typically update this (https://github.com/Wireless-Innovation-Forum/Spectrum-Access-System/blob/master/src/harness/README.md) readme, not this file.

Reply:
moved as suggested. thanks